frontend/src/App.js

```diff
@@ -37,6 +38,8 @@ class App extends Component {
       .catch(() => {
         this.setState({ isLoggedIn: false, user: this.guestUser });
       });
+
+    actions.checkLocalStorage.call(this);
```

Comment:
Don't do this please dude, see associated comment in actions where you call this.setState